### PR TITLE
[001]-counter-app

### PR DESCRIPTION
**Pull request by sakib276**
## Project Name:
[001]-Counter App

### Description
Added increasing and decreasing button
also added an option for overflowing and not take negative values

### QA Steps

- [ ] Click + count increase
- [ ] click - count decrease

### CODE REVIEW

File: 001-counter-app/Sakib 2.2/script.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+console.log(" nazmus sakib");
```
Comment (thanwin84) [inline 001-counter-app/Sakib 2.2/script.js:1]: remove this console.

File: 001-counter-app/Sakib 2.2/script.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+var count =0;
+function setapplicationtitle()
+{
+ const titleElement =document.getElementById('title');
+console.log(titleElement);
```
Comment (thanwin84) [inline 001-counter-app/Sakib 2.2/script.js:6]: Can we also delete this console log?

File: 001-counter-app/Sakib 2.2/script.js (hunk 1 above)
Comment (sakib276) [reply 001-counter-app/Sakib 2.2/script.js:1]: Obviously...
I keep it for recognizing my Code ....

**Review by thanwin84 — APPROVED** [review]

**Review by thanwin84 — APPROVED** [review]
It works  perfectly.

**Pull request merged**